src/crafting_gui.cpp

```diff
@@ -1693,6 +1706,24 @@ static bool query_is_yes( const std::string &query )
            subquery == _( "yes" );
 }
 
+static size_t craft_info_width( const size_t window_width )
+{
+    size_t reason_width = 0;
+    //The crafting speed string is necessary.  Find the longest one
+    for( const auto &pair : normalized_craft_reasons ) {
+        if( utf8_width( pair.second, true ) > int( reason_width ) ) {
+            reason_width = utf8_width( pair.second, true );
+        }
```

Comment:
```suggestion
        reason_width = std::max( utf8_width( pair.second, true ), reason_width );
```
This works if you change `reason_width` to a signed integer.